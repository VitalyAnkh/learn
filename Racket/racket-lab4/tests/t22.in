(union '(2 7 1 8) '(3 1 4))
(union '(left right) '(right wrong))
(union '(green) (union '(eggs) '(and ham)))
(union '(how much wood) (union '(would a woodchuck chuck) (union'(if a woodchuck) '(could chuck wood))))
