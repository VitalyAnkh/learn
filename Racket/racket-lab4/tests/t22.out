'(8 7 2 3 1 4)
'(left right wrong)
'(green eggs and ham)
'(much how would woodchuck a if could chuck wood)
